content/r-package-dev/debugging.md

```diff
@@ -1,9 +1,8 @@
 ---
 author: David Watkins
-date: 9999-04-30
+date: 9999-10-15
 slug: debugging
 title: Debugging
-draft: True
```

Comment:
I *think* because of the `USGSmarkdowntemplates::hugoTraining` it's automatically not putting it as draft. I also think @lindsaycarr was going to change that at some point, but has been on travel for weeks now, maybe longer? :small_airplane: